www/__tests__/metHelper.test.ts

```diff
@@ -0,0 +1,40 @@
+import { getMet } from '../js/metrics/metHelper';
+import { mockBEMUserCache } from '../__mocks__/cordovaMocks';
+import { mockLogger } from '../__mocks__/globalMocks';
+import fakeLabels from '../__mocks__/fakeLabels.json';
+import { getConfig } from '../js/config/dynamicConfig';
+import { initCustomDatasetHelper } from '../js/metrics/customMetricsHelper';
+
+mockBEMUserCache();
+mockLogger();
+
+global.fetch = (url: string) =>
+  new Promise((rs, rj) => {
+    setTimeout(() =>
+      rs({
+        text: () =>
+          new Promise((rs, rj) => {
+            let myJSON = JSON.stringify(fakeLabels);
+            setTimeout(() => rs(myJSON), 100);
+          }),
+      }),
+    );
+  }) as any;
+
+it('gets met for mode and speed', () => {
+  expect(getMet('WALKING', 1.47523, 0)).toBe(4.3);
+  expect(getMet('BICYCLING', 4.5, 0)).toBe(6.8);
+  expect(getMet('UNICYCLE', 100, 0)).toBe(0);
+  expect(getMet('CAR', 25, 1)).toBe(0);
+});
+
+it('gets custom met for mode and speed', async () => {
+  initCustomDatasetHelper(getConfig());
+  await new Promise((r) => setTimeout(r, 500));
```

Comment:
That would be cleaner/more proper to me, but I just tried it and it did not work - I think that's because appConfig is not the only async element of this workflow chain, it also calls `getLabelOptions` which is async.

`getLabelOptions(newConfig).then((labelOptions)`

what I can do though is set it up so that this function can be awaiting in the tests [51f6ece](https://github.com/e-mission/e-mission-phone/pull/1098/commits/51f6ece32cf73c110ce9b17477224ea2c52f95b6)

This is much cleaner in testing, and still allows us to not hold up the app for it to load during initialization (because we don't care about the timing of the actual initialization)